src/Makefile

```diff
@@ -61,3 +61,7 @@ check-physlr-index-stLFR-10X: all
 check-physlr-makebf: all
 	./physlr-makebf -t16 -k8 -b1000 -o tiny.bf data/tiny_k8.rep
 	./physlr-indexlr -t16 -k8 -w1 -r tiny.bf data/stLFR.tiny.fq | diff -q - data/stLFR.tiny.filtered.physlr.tsv.good
+
+check-physlr-overlap: all
+	./physlr-overlap -t1 -n1 data/tiny.overlap.input.tsv | sort | diff -q - data/tiny.physlr.overlap.n1.sorted.tsv
+	./physlr-overlap -t1 -n2 data/tiny.overlap.input.tsv | sort | diff -q - data/tiny.physlr.overlap.n2.sorted.tsv
```

Comment:
Perhaps if you are parallelizing, it would be good to have a test that tests using multiple threads?

Reply:
This PR has been rebased on the unit test PR. I will change it there